src/beautifiers/prettydiff.coffee

```diff
@@ -117,6 +117,7 @@ module.exports = class PrettyDiff extends Beautifier
         source: text
         lang: lang
         mode: "beautify"
+        crlf: getDefaultLineEnding() # solved issue 1457
```

Comment:
Yes, that is the correct option name and yes `eol` is a more intelligent name.  http://prettydiff.com/documentation.xhtml#crlf